package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "xrpl",
-  "version": "2.0.0-beta.0",
+  "version": "2.0.0-beta.1",
```

Comment:
Should this be changing in this PR?

Reply:
This is changing because @mvadari  and I forgot to run `npm` on the PR that changed that `version`

It's autogenerated and shouldn't be a problem.